nexus_constructor/main_window.py

```diff
@@ -27,11 +29,22 @@
     LinkTransformation,
 )
 from nexus_constructor.json import filewriter_json_writer
+from nexus_constructor.json.filewriter_json_reader import json_to_nexus
 
 NEXUS_FILE_TYPES = {"NeXus Files": ["nxs", "nex", "nx5"]}
 JSON_FILE_TYPES = {"JSON Files": ["json", "JSON"]}
 
 
+def show_warning_dialog(message: str, title: str, additional_info: Optional[str]):
+    msg = QMessageBox()
+    msg.setIcon(QMessageBox.Warning)
+    msg.setText(message)
+    msg.setInformativeText(additional_info)
+    msg.setWindowTitle(title)
+    msg.setStandardButtons(QMessageBox.Ok)
+    msg.exec_()
```

Comment:
think `msg.show()` will work here.

Reply:
It doesn't, that's what I tried first.

Reply:
Ahah, if I set the parent then `show()` works.